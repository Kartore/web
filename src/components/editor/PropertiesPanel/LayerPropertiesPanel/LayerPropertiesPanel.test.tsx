import { composeStories } from '@storybook/react';
import { render } from '@testing-library/react';
import { describe } from 'vitest';

import * as Stories from './LayerPropertiesPanel.stories.tsx';

const { Default } = composeStories(Stories);

describe('Component: PropertiesPanel', () => {
  describe('Snapshot', () => {
    it('Default', () => {
      const { asFragment } = render(<Default />);
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
