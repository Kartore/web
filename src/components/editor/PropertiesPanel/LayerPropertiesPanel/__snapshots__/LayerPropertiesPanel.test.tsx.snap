// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Component: PropertiesPanel > Snapshot > Default 1`] = `
<DocumentFragment>
  <div
    class="px-4"
  >
    <div
      aria-labelledby="react-aria-:r1:"
      class="h-auto w-full horizontal"
      id="react-aria-:r0:"
      role="group"
    >
      <div
        class="flex items-center justify-between"
      >
        <label
          id="react-aria-:r1:"
        >
          Zoom Level Range
        </label>
        <output
          aria-live="off"
          for="react-aria-:r1:-0 react-aria-:r1:-1"
        >
          0 - 24
        </output>
      </div>
      <div
        class="relative mt-2 h-0.5 w-full rounded-full bg-gray-300"
        style="position: relative; touch-action: none;"
      >
        <div
          class="absolute h-0.5 bg-gray-500"
          style="left: 0%; right: 0%;"
        />
        <div
          class="thumb mt-[1px] size-3 rounded-full border border-gray-500 bg-white"
          style="position: absolute; left: 0%; transform: translate(-50%, -50%); touch-action: none;"
        >
          <div
            style="border: 0px; clip-path: inset(50%); height: 1px; margin: -1px; overflow: hidden; padding: 0px; position: absolute; width: 1px; white-space: nowrap;"
          >
            <input
              aria-describedby=""
              aria-details=""
              aria-labelledby="react-aria-:r1:"
              aria-orientation="horizontal"
              aria-valuetext="0"
              id="react-aria-:r1:-0"
              max="24"
              min="0"
              step="1"
              tabindex="0"
              type="range"
              value="0"
            />
          </div>
        </div>
        <div
          class="thumb mt-[1px] size-3 rounded-full border border-gray-500 bg-white"
          style="position: absolute; left: 100%; transform: translate(-50%, -50%); touch-action: none;"
        >
          <div
            style="border: 0px; clip-path: inset(50%); height: 1px; margin: -1px; overflow: hidden; padding: 0px; position: absolute; width: 1px; white-space: nowrap;"
          >
            <input
              aria-describedby=""
              aria-details=""
              aria-labelledby="react-aria-:r1:"
              aria-orientation="horizontal"
              aria-valuetext="24"
              id="react-aria-:r1:-1"
              max="24"
              min="0"
              step="1"
              tabindex="0"
              type="range"
              value="24"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
